conbench/entities/benchmark_result.py

```diff
@@ -57,8 +58,41 @@ def create(data):
         tags = data["tags"]
         has_error = "error" in data
 
+        stats_fields = [
+            "iterations",
+            "mean",
+            "median",
+            "min",
+            "max",
+            "stdev",
+            "q1",
+            "q3",
+            "iqr",
+        ]
         if has_error:
             benchmark_result_data = {"error": data["error"]}
+        # calculate stats if any missing
+        # explicit `is None` because `stdev` is often 0
+        elif any([data["stats"].get(field) is None for field in stats_fields]):
+            dat = data["stats"]["data"]
+            fmt = "{:.6f}"  # not really sure why we're doing this, but now here for backwards consistency
```

Comment:
I'm happy to not round everything to 6 decimal places and convert numbers to strings if we're ok with it; this is just what we were doing before for whatever reason

Reply:
Oooph, yeah IMO not rounding, or at least not rounding to the point of JSON precision would be nice